test/unit/instrumentation/core/fs.test.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const crypto = require('node:crypto')
+const path = require('node:path')
+const os = require('node:os')
+const helper = require('../../../lib/agent_helper')
+const { removeModules } = require('../../../lib/cache-buster')
+
+const nodeMajor = parseInt(process.version.split('.')[0].replace('v', ''), 10)
+
+tap.beforeEach((t) => {
+  t.context.agent = helper.instrumentMockedAgent()
+  t.context.fs = require('node:fs')
+})
+
+tap.afterEach((t) => {
+  helper.unloadAgent(t.context.agent)
+  removeModules(['node:fs'])
+})
+
+tap.test('stat method gets instrumented', (t) => {
```

Comment:
we already have tests in `tests/integration/fs.tap.js`. I would add the glob tests there and remove this